External/FEXCore/Source/Interface/HLE/Thunks/Thunks.cpp

```diff
@@ -328,13 +339,13 @@ namespace FEXCore {
 
             const auto InitSym = std::string("fexthunks_exports_") + Name;
 
-            ExportEntry* (*InitFN)();
+            ExportEntry* (*InitFN)(uintptr_t, uintptr_t);
             (void*&)InitFN = dlsym(Handle, InitSym.c_str());
             if (!InitFN) {
                 ERROR_AND_DIE_FMT("LoadLib: Failed to find export {}", InitSym);
             }
 
-            auto Exports = InitFN();
+            auto Exports = InitFN((uintptr_t)MakeHostTrampolineForGuestFunction, (uintptr_t)FinalizeHostTrampolineForGuestFunction);
```

Comment:
Maybe  make it a struct here?

Reply:
New version uses FEX symbols directly, which allows for keeping the shared set of interfaces more easily in sync.

Reply:
Hmm. How would you feel about FEX exposing a data symbol with function pointers inside? Makes it easier for versioning.

Reply:
Where do we need things to be versioned?